src/test/regress/expected/multi_alter_table_add_constraints_without_name.out

```diff
@@ -0,0 +1,202 @@
+ products_pkey
+(1 row)
+
+-- Check that the primary key name created on the coordinator is sent to workers and
+-- the constraints created for the shard tables conform to the <conname>_shardid scheme.
+\c - - :public_worker_1_host :worker_1_port
+SELECT con.conname
+    FROM pg_catalog.pg_constraint con
+      INNER JOIN pg_catalog.pg_class rel ON rel.oid = con.conrelid
+      INNER JOIN pg_catalog.pg_namespace nsp ON nsp.oid = connamespace
+		WHERE rel.relname = 'products_5410000';
+        conname
+---------------------------------------------------------------------
+ products_pkey_5410000
+(1 row)
+
+\c - - :master_host :master_port
+ALTER TABLE sc1.products DROP CONSTRAINT products_pkey;
+ALTER TABLE sc1.products ADD PRIMARY KEY(product_no);
+DROP TABLE sc1.products;
+-- Check "ADD PRIMARY KEY" with reference table
+CREATE TABLE sc1.products_ref (
+    product_no integer,
+    name text,
+    price numeric
+);
+CREATE TABLE sc1.products_ref_2 (
+	    product_no integer,
+	    name text,
+	    price numeric
+);
+CREATE TABLE sc1.products_ref_3 (
+            product_no integer,
+            name text,
+            price numeric
+);
+SELECT create_reference_table('sc1.products_ref');
+ create_reference_table
+---------------------------------------------------------------------
+
+(1 row)
+
+SELECT create_reference_table('sc1.products_ref_3');
+ create_reference_table
+---------------------------------------------------------------------
+
+(1 row)
+
+-- Check for name collisions
+ALTER TABLE sc1.products_ref_3 ADD CONSTRAINT products_ref_pkey PRIMARY KEY(name);
+ALTER TABLE sc1.products_ref_2 ADD CONSTRAINT products_ref_pkey1 PRIMARY KEY(name);
+ALTER TABLE sc1.products_ref ADD PRIMARY KEY(name);
+SELECT con.conname
+    FROM pg_catalog.pg_constraint con
+      INNER JOIN pg_catalog.pg_class rel ON rel.oid = con.conrelid
+      INNER JOIN pg_catalog.pg_namespace nsp ON nsp.oid = connamespace
+              WHERE rel.relname = 'products_ref';
+      conname
+---------------------------------------------------------------------
+ products_ref_pkey2
+(1 row)
+
+ALTER TABLE sc1.products_ref DROP CONSTRAINT products_ref_pkey2;
+DROP TABLE sc1.products_ref;
+-- Check with max table name (63 chars)
+CREATE TABLE sc1.verylonglonglonglonglonglonglonglonglonglonglonglonglonglonglonger (
+	            product_no integer,
+	            name text,
+                    price numeric
+		);
+NOTICE:  identifier "verylonglonglonglonglonglonglonglonglonglonglonglonglonglonglonger" will be truncated to "verylonglonglonglonglonglonglonglonglonglonglonglonglonglonglon"
+SELECT create_distributed_table('sc1.verylonglonglonglonglonglonglonglonglonglonglonglonglonglonglon', 'product_no');
+ create_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
+ALTER TABLE sc1.verylonglonglonglonglonglonglonglonglonglonglonglonglonglonglon ADD PRIMARY KEY(product_no);
```

Comment:
could be nice to also force a name collision, e.g. find two different names that would result in the same primary key name when truncing and adding _pkey.

(this seems to work correctly, the second one uses _pkey1)